.github/workflows/smoke-test.yaml

```diff
@@ -37,64 +37,6 @@ jobs:
         run: |
           make -C install/kubernetes lint
 
-  quick-install:
-    strategy:
-      matrix:
-        target:
-          - name: quick-install
-            template: install/kubernetes/quick-install.yaml
-          - name: experimental-install
-            template: install/kubernetes/experimental-install.yaml
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Check ${{ matrix.target.template }}
-        run: |
-          make -C install/kubernetes ${{ matrix.target.name }}
-          git diff --exit-code
```

Comment:
I think we can still keep this section where we check if the `quick-install/experimental-install` is not outdated?

Reply:
agreed with @fristonio. 

I am also thinking about spin off this check, `preflight-clusterrole` and `lint` to another file (e.g. helm-check.yaml).